CMakeLists.txt

```diff
@@ -15,9 +15,12 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON) # -fPic
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS TRUE)
 
+enable_testing()
```

Comment:
This only works with ctest e.g. if you use add_test, so this line should be useless/can be removed again.